core-bundle/assets/controllers/check-all-controller.js

```diff
@@ -3,7 +3,61 @@ import { Controller } from '@hotwired/stimulus';
 export default class extends Controller {
     static targets = ['source', 'input'];
 
-    update() {
+    static values = {
+        rowGuards: {
+            type: Array,
+            default: ['input', 'label', 'button', 'a', '.operations'],
+        },
+    };
+
+    initialize() {
+        this.start = null;
+    }
```

Comment:
this is just a class variable that could be defined as such, right?

Reply:
I.e. extending from the controller, you would be able to modify the class variable but that's not what we want as this should only live in this instance (as a variable). It's simply reserving space to save the current target.

Since there isn't really a constructor to define it, I initialize it right after the controller is instantiated.

Do you propose using a stimulus `value` instead (as this should definitely not be accessed from outside nor any other controllers inheriting from this one)?

Reply:
See 306e1dcff713c69cec78d297e26a201611a5b6ec